### ISSUE TITLE
json files referenced in nist_data/README.md no longer available?

### ISSUE THREAD

**Issue by mbanck** [MolSSI/QCElemental#158]
I tried to download the json files https://nist.gov/srd/srd_data//srd121_allascii_2014.json and https://nist.gov/srd/srd_data/srd144_Atomic_Weights_and_Isotopic_Compositions_for_All_Elements.json but those just return a regular HTML page, do I need to do something special to get the JSON?

**Comment by loriab** [comment]
Agree. When I tried to access the JSON a few weeks ago, it was gone. Gov't access back to ascii or webform :-( I figured that it was best to keep original links for historical reference.

Same for the next thing to harvest, the codata2018, for which https://pml.nist.gov/cuu/Constants/Table/allascii.txt seems the most compact and collectable source.
